src/snapred/ui/workflow/ReductionWorkflow.py

```diff
@@ -171,14 +184,113 @@ def _triggerReduction(self, workflowPresenter):
                 if unfocusedData is not None:
                     self.outputs.append(unfocusedData)
 
+            elif isinstance(response.data, ArtificialNormResponse):
+                self._artificialNormalizationView.updateRunNumber(runNumber)
+                self._artificialNormalization(workflowPresenter, response.data, runNumber)
+                return self.responses[-1]
             # Note that the run number is deliberately not deleted from the run numbers list.
             # Almost certainly it should be moved to a "completed run numbers" list.
 
         # SPECIAL FOR THE REDUCTION WORKFLOW: clear everything _except_ the output workspaces
         #   _before_ transitioning to the "save" panel.
         # TODO: make '_clearWorkspaces' a public method (i.e make this combination a special `cleanup` method).
         self._clearWorkspaces(exclude=self.outputs, clearCachedWorkspaces=True)
+        workflowPresenter.advanceWorkflow()
+        return self.responses[-1]
+
+    def _artificialNormalization(self, workflowPresenter, responseData, runNumber):
+        view = workflowPresenter.widget.tabView  # noqa: F841
+        try:
+            # Handle artificial normalization request here
+            request_ = CreateArtificialNormalizationRequest(
+                runNumber=runNumber,
+                useLiteMode=self._reductionRequestView.liteModeToggle.field.getState(),
+                peakWindowClippingSize=int(self._artificialNormalizationView.peakWindowClippingSize.field.text()),
+                smoothingParameter=self._artificialNormalizationView.smoothingSlider.field.value(),
+                decreaseParameter=self._artificialNormalizationView.decreaseParameterDropdown.currentIndex() == 1,
+                lss=self._artificialNormalizationView.lssDropdown.currentIndex() == 1,
+                diffractionWorkspace=responseData.diffractionWorkspace,
+            )
+            response = self.request(path="reduction/artificialNormalization", payload=request_)
+            # Update workspaces in the artificial normalization view
+            diffractionWorkspace = responseData.diffractionWorkspace
+            artificialNormWorkspace = response.data
+
+            if diffractionWorkspace and artificialNormWorkspace:
+                self._artificialNormalizationView.updateWorkspaces(diffractionWorkspace, artificialNormWorkspace)
+            else:
+                print(f"Error: Workspaces not found in the response: {response.data}")
+        except Exception as e:  # noqa: BLE001
```

Comment:
Please dont swallow errors like this, let it fail.  Its much harder to track down without a stacktrace, or rewrap it with a human readable error message and rethrow.